fsspec/asyn.py

```diff
@@ -61,8 +61,10 @@ def sync(loop, func, *args, timeout=None, **kwargs):
     """
     Make loop run coroutine until it returns. Runs in other thread
 
-    Example usage:
-        fsspec.asyn.sync(fsspec.asyn.get_loop(), func, *args, timeout=timeout, **kwargs)
+    Examples
+    --------
+    >>> fsspec.asyn.sync(fsspec.asyn.get_loop(), func, *args,
+                         timeout=timeout, **kwargs)
```

Comment:
This was fun, an isolated asterisk in a docstring is interpreted as the start of emphasis. You can prepend a backslash and sphinx is happy but flake8 is not. Converting it to an actual code block removes the problem, but then the line length is too long so I have split it into 2 lines.